src/store/acquisitionPackage/index.ts

```diff
@@ -1924,13 +1932,105 @@ export class AcquisitionPackageStore extends VuexModule {
           : await api.contractTypeTable.create(data);
       this.setContractType(savedContractType);
       this.setAcquisitionPackage({
-        ...this.contractType,
+        ...this.acquisitionPackage,
         contract_type: sys_id,
       } as AcquisitionPackageDTO);
     } catch (error) {
       throw new Error(`error occurred saving PoP data ${error}`);
     }
   }
+
+  /**
+   * A method to retrieve data from tables that are used strictrly for attaching files

```

Comment:
misspelling in comment `strictrly` but I do like that word :)

Reply:
resolved